src/main/java/io/goobox/sync/storj/App.java

```diff
@@ -151,6 +154,11 @@ private void init(boolean resetAuthFile) {
         storj.setConfigDirectory(Utils.getDataDir().toFile());
         storj.setDownloadDirectory(syncDir.toFile());
 
+        if (!storj.keysExist()) {
+            logger.error("keyfile not found at the expected dir " + Utils.getDataDir().toFile());
```

Comment:
It would be better to improve the existing log message at line 276 than adding a new one here.